src/nbl/asset/utils/CHLSLCompiler.cpp

```diff
@@ -57,7 +59,10 @@ static tcpp::IInputStream* getInputStreamInclude(
     uint32_t _maxInclCnt,
     const char* _requesting_source,
     const char* _requested_source,
-    bool _type // true for #include "string"; false for #include <string>
+    bool _type, // true for #include "string"; false for #include <string>
```

Comment:
name this "isRelative" also remove the weird `_` before var names and make them consistent (camelCase)

see how shaderc names them:
```c++
// The kinds of include requests.
enum shaderc_include_type {
  shaderc_include_type_relative,  // E.g. #include "source"
  shaderc_include_type_standard   // E.g. #include <source>
};
```